x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_form.tsx

```diff
@@ -44,20 +42,17 @@ export const RelevanceTuningForm: React.FC = () => {
   return (
     <section className="relevanceTuningForm">
       <form>
-        <EuiPageHeader>
-          <EuiPageHeaderSection>
-            <EuiTitle size="m">
-              <h2>
-                {i18n.translate(
-                  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.manageFields.title',
-                  {
-                    defaultMessage: 'Manage fields',
-                  }
-                )}
-              </h2>
-            </EuiTitle>
-          </EuiPageHeaderSection>
-        </EuiPageHeader>
+        <EuiTitle size="m">
+          <h2>
+            {i18n.translate(
+              'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.manageFields.title',
+              {
+                defaultMessage: 'Manage fields',
+              }
+            )}
+          </h2>
+        </EuiTitle>
+        <EuiSpacer />
```

Comment:
[not blocking, minor UI request/can be done later] There's two `EuiSpacer`s in a row now when the filter field input isn't present - any chance we can move the 2nd `EuiSpacer` to within the `{schemaFields.length > FIELD_FILTER_CUTOFF && (` conditional?

<img width="641" alt="" src="https://user-images.githubusercontent.com/549407/109745613-ff10ec80-7b88-11eb-96f3-cfdb785a35b1.png">

Reply:
408984a806fafb58e525e178d6d87d7ea8c3ea43